src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LANGTH = 'langth';
+
+interface Filter<T> {
+  sortField: keyof T | '';
+  reversed: boolean;
+}
+
+function gooods<T>(
```

Comment:
The function name `gooods` seems to be misspelled. It should likely be `goods` to match the context of the application.